io.github.jliljebl.Flowblade.yaml

```diff
@@ -139,22 +125,6 @@ modules:
         mirror-urls:
           - http://http.debian.net/debian/pool/main/s/swig/swig_3.0.12.orig.tar.gz
 
-  - name: jack2
```

Comment:
The audio-filter module from MLT needs jack.

Reply:
I'm putting this back in now